Argo/Types/JSON.swift

```diff
@@ -28,6 +28,15 @@ public extension JSON {
   }
 }
 
+extension JSON {
+  func jsonForKey(key: Swift.String) -> JSON {
```

Comment:
I feel like getting "object" into the name for this function would be useful. Maybe `parseObject` or something like that? It's like `decodeObject` except it doesn't follow through and map out the keys.